requirements/edx/testing.txt

```diff
@@ -992,7 +992,7 @@ optimizely-sdk==4.1.1
     # via
     #   -c requirements/edx/../constraints.txt
     #   -r requirements/edx/base.txt
-ora2==6.0.31
+ora2 @ git+https://github.com/open-craft/edx-ora2@yusuf-musleh/serialize-tag-data-in-openassessment-olx
```

Comment:
Can you remove this change from the diff for now? We can merge them independently.